scipy/stats/tests/test_bootstrap.py

```diff
@@ -54,6 +58,68 @@ def statistic(x, y, axis):
     with pytest.raises(ValueError, match=message):
         bootstrap(([1, 2, 3],), np.mean, random_state='herring')
 
+@pytest.mark.parametrize("method", ['basic', 'percentile', 'BCa'])
+def test_bootstrap_paired(method):
+    # test that `paired` works as expected
+    np.random.seed(0)
+    n = 100
+    x = np.random.rand(n)
+    y = np.random.rand(n)
+
+    def my_statistic(x, y, axis=-1):
+        return ((x-y)**2).mean(axis=axis)
+
+    def my_paired_statistic(i, axis=-1):
+        a = x[i]
+        b = y[i]
+        res = my_statistic(a, b)
+        return res
```

Comment:
Blank line after the definiton:
```suggestion
        return res

```

Reply:
I'll run flake8 locally. In my haste I let that slide.